### DISCUSSION TITLE
Audio changes

### DISCUSSION THREAD

**Discussion by rosetown-kelly** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#485] [General]
Pulled my tub out of storage for the winter and updated to the latest firmware. Love the web interface changes and the web firmware update option. Thanks so much. 

2 questions though. 

Is there any way to bring the startup chime back? 

And is there any way to revert to the old simple button press sounds?

**Comment by visualapproach** [comment]
Thx

> Is there any way to bring the startup chime back? 

Hehe I may have to enable it again  .. The sound routine is reworked. Until further you can enjoy 'furelise.tune' if you're on beta version. (From spa config menu. Manual will be updated later)

> And is there any way to revert to the old simple button press sounds? 

I hear you. Now they are hardcoded so I must make a few changes to enable customizable sounds
